src/plugins/discover/public/components/view_mode_toggle/view_mode_toggle.tsx

```diff
@@ -36,7 +35,7 @@ export const DocumentViewModeToggle = ({
   }
 
   return (
-    <EuiTabs size="s" css={tabsCss} data-test-subj="dscViewModeToggle" bottomBorder={false}>
+    <EuiTabs size="s" css={tabsCss} data-test-subj="dscViewModeToggle" bottomBorder>
```

Comment:
I think `bottomBorder` is true by default, so you can probably remove the prop altogether.

```suggestion
    <EuiTabs size="s" css={tabsCss} data-test-subj="dscViewModeToggle">
```